### DISCUSSION TITLE
argo cli tools for ArgoCD, ArgoWorkflow

### DISCUSSION THREAD

**Discussion by tiwarisanjay** [actions/virtual-environments#4528] [Installed software]
Hi I was hoping if we can have some argo related libraries on Agents  as we might be running some ArgoWorkflow or Triggering Argo Events or Running some cli commands while running ADO pipelines.

**Comment by miketimofeev** [comment]
Hi @tiwarisanjay!
There is an action exists that probably suits your needs https://github.com/clowdhaus/argo-cd-action

**Reply by miketimofeev** [reply]
And to install it in ADO you can use the following steps, it takes only 2 seconds to install the CLI
```
# Download the binary
curl -sLO https://github.com/argoproj/argo-workflows/releases/download/v3.2.3/argo-linux-amd64.gz

# Unzip
gunzip argo-linux-amd64.gz

# Make binary executable
chmod +x argo-linux-amd64

# Move binary to path
mv ./argo-linux-amd64 /usr/local/bin/argo

# Test installation
argo version
```
